server/src/callback.js

```diff
@@ -1,20 +1,37 @@
-function eventCallback(req, res) {
-  // check auth
-  if (req.header('Bearer')) {
-    console.log(req.header('Bearer'));
-    console.log(req.header('Authorization'));
-  }
+const SESCallback = {
+  creator: undefined,
+  eventCallback(req, res) {
+    const job = {
+      name: req.body.JobType.Name,
+      identifier: req.body.Identifier,
+      description: req.body.JobType.Description,
+    };
 
-  console.log(req.body);
+    Promise.resolve(this.creator.location({
+      name: req.body.Address.GnafId,
```

Comment:
this would be 'scene' IMO. as in, when someone uses the app and says where they are going I see most jobs having "scene" and "lhq" as the default two.

LHQ would be the ords under EntityAssignedTo and could be added as well.

Reply:
👍